content/en/docs/setup/best-practices/certificates.md

```diff
@@ -162,4 +198,11 @@ These files are used as follows:
 | controller-manager.conf | kube-controller-manager | Must be added to manifest in `manifests/kube-controller-manager.yaml` |
 | scheduler.conf          | kube-scheduler          | Must be added to manifest in `manifests/kube-scheduler.yaml`          |
 
+The following illustrates the files listed in the previous table that contain client user certificates and keys:
```

Comment:
nit: The following ?

Reply:
Okay. I updated it to say: "The following files illustrate full paths to the files listed in the previous table:"